src/System.Runtime/tests/System/NullableMetadataTests.cs

```diff
@@ -39,9 +39,8 @@ public static void ShimsDontHaveNullablePublicOnlyAttribute()
             Assert.Empty(module.CustomAttributes.Where(a => a.GetType().FullName.Equals(NullablePublicOnlyAttributeFullName)));
         }
 
-        private static bool NullableAttributesFilter(Attribute attribute)
-        {
-            return attribute.GetType().FullName.Equals(NullableAttributeFullName) || attribute.GetType().FullName.Equals(NullableContextAttributeFullName);
-        }
+        private static bool NullableAttributesFilter(Attribute attribute) =>
```

Comment:
I meant to convert it to a local function inside the method where it's used 😅 but a nice improvement still

Reply:
LOL, I wasn't sure cause inline is using `=>` . I'm not a big fan of local functions. So I rather just have this code directly in the `Linq` lambda. Will do that instead.

Reply:
Fixed.